ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/resetbutton/v1/resetbutton/resetbutton.html

```diff
@@ -0,0 +1,34 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.button="com.adobe.cq.forms.core.components.models.form.Button"
```

Comment:
do we need to have this sightly here? I think it can be inherited from button components.

Reply:
Okay will remove this file. The only difference between this and the button sightly is the `data-cmp-is` field, which is `adaptiveFormResetButton` and `adaptiveFormSubmitButton`. If we rely on inheritance then both of them will be `adaptiveFormButton`. I hope that's okay.